SYS_ATL/LoopIR_unification.py

```diff
@@ -701,10 +701,13 @@ def get_arg(fa):
                                      stmt_block[0].srcinfo)
             elif fa.type == T.bool:
                 if fa.name in self.bool_holes:
-                    return self.bool_holes[fa.name]
+                    if self.bool_holes[fa.name] is False:
+                        return LoopIR.Const(False, T.bool, stmt_block[0].srcinfo)
+                    else:
+                        return self.bool_holes[fa.name]
                 else:
-                    # doesn't matter; argument un-used
-                    return LoopIR.Const(True, T.bool, stmt_block[0].srcinfo)
+                    # subprocedure argument must be consistent
+                    assert False, "bad case"
```

Comment:
@gilbo I'm not sure what was the original intention of this case? `self.bool_holes` is constructed by `subproc.args`, and this function `get_arg` is called with `subproc.args`, so there should be no case where `fa.name` is not in `self.bool_holes`.